psi-ms.obo

```diff
@@ -45,6 +45,7 @@ name: has_units
 [Typedef]
 id: part_of
 name: part_of
+xref: BFO:0000050
```

Comment:
@matentzn Can you update the version of the ontology. See the failing Actions https://github.com/HUPO-PSI/psi-ms-CV/actions/runs/7638645173/job/20810093289

Reply:
Feel free to close this PR and redo it, I just wanted to show you how you can map your part of to RO, so it displays correctly in OLS and interoperates with other ontologies. You can also directly edit my PR and make the required changes!

Reply:
Done. Thanks a lot for the tip.